### ISSUE TITLE
[BUG]:  Sending E-Mail does not work on Shinyapps.io Server

### ISSUE THREAD

**Issue by CloudStriderGames** [datawookie/emayili#113]
### Description of the bug

Hello, sadly I don't have much of an Error log, but sending an email works if I run the App locally in RStudio, but if I try the same functionality when the app is hosted on ShinyApps.io it crashes after 5 attempts. Might this be something with my gmail security settings? Could there be another reason.

Sorry for this shoddy Report, but on the server there is very little of an error log.
Basically just that the app shut down after 5 attempts.

### Steps To Reproduce

Build an app that sends an email. (using a gmail account. Might have to adjust the settings to allow access from less secure places)
Host it on ShinyApps.io.
Try it out.

**Comment by datawookie** [comment]
Hi @CloudStriderGames, you should find that `{emayili}` works well within a Shiny app. I suspect that the problem lies in how you are managing your cedentials. I've added [a short post](https://datawookie.dev/blog/2022/01/emayili-sending-email-from-shiny/) on my blog which should hopefully help. Let me know if you still have trouble.

> Might have to adjust the settings to allow access from less secure places).

If you have adjusted this setting in your Gmail account and are able to send emails locally using `{emayili}` then this should not be an issue.

Best regards, Andrew.
